BYTETRACK:
  TRACK_THRESH: 0.35
  MATCH_THRESH: 0.9
  MIN_BOX_AREA: 10
  TRACK_BUFFER: 30
  FRAME_RATE: 30
